README.md

```diff
@@ -98,6 +98,8 @@ Beautifier Options:
   -k, --keep-array-indentation      Preserve array indentation
   -x, --unescape-strings            Decode printable characters encoded in xNN notation
   -w, --wrap-line-length            Wrap lines at next opportunity after N characters [0]
+  -a, --wrap-attributes             Wrap attributes to new lines
```

Comment:
Instead of a true/false, please make this a string valid options for now are "auto" (default) and "force".  This will leave it open for us to add "none" or other option later.